sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractModuleConfiguration.java

```diff
@@ -137,7 +140,7 @@ private static boolean validateOutputDir(Path analyzerOutputDir) {
   }
 
   private static DirectoryStream.Filter<Path> protoFileFilter() {
-    return p -> p.getFileName().toString().toLowerCase().endsWith(".pb");
+    return p -> p.getFileName().toString().toLowerCase(Locale.ROOT).endsWith(".pb");
```

Comment:
out of curiosity - why was this Locale.ROOT needed?

Reply:
It was a code smell fix for [S1449](https://rules.sonarsource.com/java/RSPEC-1449).

`ROOT` seems as the best option to get [neutral locale](https://docs.oracle.com/javase/7/docs/api/java/util/Locale.html#ROOT)